src/test/java/org/folio/rest/support/http/ResourceClient.java

```diff
@@ -1,5 +1,6 @@
 package org.folio.rest.support.http;
 
+import static org.folio.rest.api.TestBaseWithInventoryUtil.mockServer;
```

Comment:
I think it's a bad solution to use a variable that is initialized by Wiremock framework in the client, which could be used when Wiremock and TestBaseWithInventoryUtil are not used in the test.

I would suggest to set header somewhere in tests implicitly or create some special resource client for holdings that will override basic client methods with logic we need to add to it. 
Wiremock url could be passed in constructor for such client or it could be null in case there is no wiremock